### PR TITLE
Tuan playback/thumbnail

### PR DESCRIPTION
**Pull request by Tuanthai4444**
Fixed up some documentation to fit our needs. Also finished the slider for playback.

### CODE REVIEW

File: Source/MixdownFolder.cpp [hunk 1]
```diff
@@ -41,6 +42,16 @@ MixdownFolderComp::MixdownFolderComp(juce::AudioDeviceManager& adm) : deviceMana
     //Lambda captures event on button click and calls function
     prevButton.onClick = [this] {prevButtonClickResponse(); };
 
+    addAndMakeVisible(&audioPositionSlider);
+    audioPositionSlider.setRange(0, 0.1);
```
Comment (nlstrait) [inline Source/MixdownFolder.cpp:58]: Is this range arbitrary since we'll set a new one when loading in a mixdown?

**Review by nlstrait — CHANGES_REQUESTED** [review]
Implementation looks good on the surface, but user tests reveal a couple of issues:
1. The slider value does not change as the track is played, so you can pick a starting position, but the slider stays in that position while playing, which is a bit unintuitive.
2. The stop button/function no longer works. It only resets playback to the start of the mixdown.

**Comment by jaygrinols** [comment]
When there's only one mixdown in the selected folder, you can still press next/previous buttons. When navigating to a "blank" mixdown selection using these buttons in this state, it'll still play the previously selected mixdown. Might need to fix some functionality around there.

File: Source/MixdownFolder.cpp (hunk 1 above)
Comment (Tuanthai4444) [reply Source/MixdownFolder.cpp:58]: Yeah, range is arbitrary and we can instead set the slider to disabled if that's what we want?

**Comment by nlstrait** [comment]
> When there's only one mixdown in the selected folder, you can still press next/previous buttons. When navigating to a "blank" mixdown selection using these buttons in this state, it'll still play the previously selected mixdown. Might need to fix some functionality around there.

I believe these issues are not specific to this branch / pull request but were here before. I have fixed some if not all of these issues in the branch I am currently working in.

**Comment by Tuanthai4444** [comment]
We went with a different slider idea

**Comment by nlstrait** [comment]
While running through xcode, upon application close, I get an error indicating that one or more objects have leaked. This suggests some objects may need to be explicitly deleted using `delete`.

**Pull request closed without merging**